migrating_from_ocp_3_to_4/premigration-checklists-3-4.adoc

```diff
@@ -93,7 +93,7 @@ If an application uses an internal image in the `openshift` namespace that is no
 * [ ] Certificates that your application uses exist on the target cluster.
 * [ ] You have configured appropriate firewall rules on the target cluster.
 * [ ] You have correctly configured load balancing on the target cluster.
-* [ ] The namespaces on the target cluster do not duplicate namespaces that are being migrated.
+* [ ] If a namespace on the target cluster duplicates a namespace on the source cluster, the resources that already exist in the target namespace do not have the same names as the resources that are being migrated.
```

Comment:
@sbeskin-redhat @eriknelson I don't understand...does this mean that the names of the resources in the source namespace will change to something else when they are migrated? An example here would be very helpful.

Reply:
@stoobie No, the user is supposed to check that there is no conflict between the names that are being migrated and the names that already exist in the target namespace.

Reply:
@sbeskin-redhat 
In light of @eriknelson's comment, I suggest something like:

Objects of the same type and name on the destination and source clusters can cause the migration to fail to ensure nothing destructive occurs. For example, if a namespace already exists on the destination cluster with the same name as a namespace being migrated from the source cluster, the migration fails.